app/views/macros/edit.html.erb

```diff
@@ -5,5 +5,14 @@
 <h1>Edit macro</h1>
 
 <section>
+  <div class="row">
+    <div class="col-lg-2 button-label">Actions</div>
+    <div class="col-lg-6">
+      <%= link_to 'Execute', execute_project_macro_path(@project, @macro), method: :post, class: 'btn btn-primary', data: { confirm: "Are you sure you have no unsaved changes and would like to execute '#{@macro.name}'?" } %>
+      <% if current_user.is_admin? %>
+        <%= link_to_delete_button([@project, @macro]) %>
```

Comment:
we have `.can#{do_stuff}` all over the place in our models and view helpers. I'll fix that in another PR